content/docs/studio/selfhosted/configuration/index.md

```diff
@@ -0,0 +1,76 @@
+# Configuration
+
+Studio has a unified YAML configuration file, often referred to as `values.yaml`
```

Comment:
```suggestion
Like most Helm charts, Studio uses a standardized unified YAML configuration file, often referred to as `values.yaml`
```

Reply:
I think it's better not to mention Helm, so that AMI users don't get confused and/or worry about the underlying stack.

Reply:
But it's impossible to avoid - no matter if we use AMI or BYOC k8s, it's a shared step - users will have to be aware of the helm and `values.yaml` file to configure studio. It's impossible to skip configuration when installing studio, no matter the infra - because of networking and SCM provider configuration at the very least, right?